Jenkinsfile_CNP

```diff
@@ -5,7 +5,7 @@ import uk.gov.hmcts.contino.GradleBuilder
 
 def type = "java"
 def product = "bulk-scan"
-def component = "pay-processor"
+def component = "payment-processor"
```

Comment:
will that not change entire infrastructure? 😳

Reply:
AAT was never built and also it is needed for helm charts. Other option is to update charts folder but I thought this will make it consistent.